packages/nc-gui/components/cell/attachment/index.vue

```diff
@@ -209,15 +212,29 @@ const rowHeight = inject(RowHeightInj, ref(1.8))
               <div class="text-center w-full">{{ item.title }}</div>
             </template>
             <div v-if="isImage(item.title, item.mimetype ?? item.type)">
-              <div class="nc-attachment flex items-center justify-center" :class="{'ml-2':active}" @click.stop="selectedImage = item">
+              <div
+                class="nc-attachment flex items-center justify-center"
+                :class="{ 'ml-2': active }"
+                @click.stop="selectedImage = item"
+              >
                 <LazyCellAttachmentImage
-                  class="max-h-[1.8rem] max-w-[1.8rem]"
+                  class="h-7 w-7"
```

Comment:
Why do u have height and width here when we are setting in the conditional class below? If u want to have a default height and width value add default condition in conditional class with required style, otherwise right now there is clash of classes